src/main.js

```diff
@@ -332,8 +332,16 @@ function grammarsFromScriptElements(optNodeOrNodeList) {
   return ns;
 }
 
-function makeRecipe(recipeFn) {
-  return recipeFn.call(new Builder());
+function makeRecipe(recipe) {
+  if (typeof recipe === 'function') {
+    return recipe.call(new Builder());
+  } else {
```

Comment:
Is it valid to call this with an argument that is neither a function nor a string? If not, consider making this an `else if` and add a `else` case that throws an error.